src/jmh/java/net/sf/jabref/benchmarks/Benchmarks.java

```diff
@@ -44,7 +44,7 @@ public void init() throws IOException, SaveException {
         Random randomizer = new Random();
         for (int i = 0; i < 1000; i++) {
             BibEntry entry = new BibEntry();
-            entry.setCiteKey("id" + i);
+            entry.setBibTeXKey("id" + i);
```

Comment:
Are you talking about the general naming of the method or about the concrete usage of `setBibTeXKey(...)` in this class? :confused: 

I think the proposed naming and reasoning of @koppor is fine. ID for internal usage - "BibTeXKey" used as in the UI...
Using additional classes for this seems to be an overkill for me.